TaskLayer/SearchTask/PostSearchAnalysisTask.cs

```diff
@@ -336,6 +281,132 @@ private void QuantificationAnalysis()
                 }
             }
 
+            //If SILAC (Pre-Quantification) preparation
+            //setup variables
+            List<SilacLabel> allSilacLabels = Parameters.SearchParameters.SilacLabels;
+            SilacLabel startLabel = Parameters.SearchParameters.StartTurnoverLabel;
+            SilacLabel endLabel = Parameters.SearchParameters.EndTurnoverLabel;
+            bool quantifyUnlabeledPeptides = Parameters.ListOfDigestionParams.Any(x => x.GeneratehUnlabeledProteinsForSilac);
+            if (Parameters.SearchParameters.SilacLabels != null)
```

Comment:
This is a place where many lines are "hit" in unit tests but it is not at all clear to me the the values computed are correct. Methods with more than 100 lines spook me.